packages/frontend/editor-ui/src/features/dataStore/types.ts

```diff
@@ -17,5 +21,16 @@ declare module '@/Interface' {
 	}
 }
 
+export const isAGGridCellType = (value: unknown): value is AGGridCellType => {
```

Comment:
I just added `type-guards.ts` files for data store. Think it's a better place for type guards